Assets/HoloToolkit/UX/Resources/Shaders/Wireframe.shader

```diff
@@ -0,0 +1,109 @@
+// Upgrade NOTE: replaced 'mul(UNITY_MATRIX_MVP,*)' with 'UnityObjectToClipPos(*)'
```

Comment:
The Toolkit has a [wireframe shader](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit/SpatialMapping/Shaders/Wireframe.shader) already. Can we reuse it?

Reply:
Yes, I believe they're identical.

Reply:
I thought I had removed it in the latest build.  I'll remove it.

thanks,
-pat